scripts/bash_utils.sh

```diff
@@ -23,14 +23,13 @@ function setup_python {
 }
 
 function install_py_packages {
+  python3 -m venv .venv
```

Comment:
Will it work for everyone?

Reply:
🤦🏻 , I ust realised that check a little bit below which I removed actually checks if it is in venv or not...